pkgdb2/lib/__init__.py

```diff
@@ -2532,6 +2532,32 @@ def edit_action_status(
             session.rollback()
             pkgdb2.LOG.exception(err)
             raise PkgdbException('Could not edit action.')
+
+        # Approve the request.branch awaiting review on this package now
+        # that it was approved
+        if action_status == 'Approved' \
+                and admin_action.action == 'request.package':
+            pkg = admin_action.info_data.get('package')
+            requests = search_actions(
+                session, package=pkg, action='request.branch',
```

Comment:
I'm thinking that this may be wrong, when you request a new package on several branches, you get several request.package
Let me fix this